libs/blocks/text/text.js

```diff
@@ -52,6 +52,32 @@ function decorateBlockIconArea(el) {
   });
 }
 
+function decorateLinkFarms(el) {
+  const { miloLibs, codeRoot } = getConfig();
+  loadStyle(`${miloLibs || codeRoot}/blocks/text/link-farms.css`);
+  const hasSpacing = el.classList.toString().includes('-spacing');
+  if (!hasSpacing) el.classList.add('xl-spacing');
```

Comment:
Nit: you could omit the 2 above lines and just apply this default in css and if the style is there it will get over-written.